infrastructure/controller/pkg/endpoints/server.go

```diff
@@ -0,0 +1,42 @@
+package endpoints
+
+import (
+	"github.com/gorilla/mux"
+	"k8s.io/klog"
+	"log"
+	"net/http"
+	"os"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+func Serve(clientSet *kubernetes.Clientset, namespace string) {
+	r := mux.NewRouter()
+
+	// Load authentication middleware only if auth env is present
+	authEnabled := false
+	systemToken := os.Getenv("systemToken")
```

Comment:
Should we put this ENV also in the deployment.yaml file, from the security configMap?

Reply:
Good catch